### DISCUSSION TITLE
look what i made

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#791] [(A) Anything]
https://teespring.com/prodigy-hacker-mask

i wonder if this will sell hmm

**Comment by ArcerionDev** [comment]
![image](https://user-images.githubusercontent.com/60945964/112175331-8de3b880-8bcd-11eb-85c9-b95253a29b95.png)

**Reply by ghost** [reply]
shit you found it

**Reply by ArcerionDev** [reply]
"chickenOnY"

**Reply by ghost** [reply]
ugh im working on it

**Comment by ArcerionDev** [comment]
and sorry but that's really bad

**Reply by ghost** [reply]
aww

**Comment by irhrhd** [comment]
Im at school rn lol

**Comment by ghost** [comment]
@ChickenOnYT What is that?

I see it as a handkerchief cuz it looks like that
